Quality of Life Update

Bongo9911 opened the pull request:
- Updated navigation on home screen to navigate through tabs rather than through stack
- Made keyboard dismiss when tapping off on home screen
- Changed spacing on home screen to space evenly
- Added experimental splash screen which shows while app loads
- Added the ability to style cards
- Changed styling of cards on Buddies page so that they’re each the same height

andrewJamesBaker reviewed (changes requested):
So a couple of things.
1. The start-up "Habit Buddy" font is not good. I would suggest finding a more professional font.

2.  The app now defaults to dark mode. I know that this is still a work in progress, but I wanted to make note of that.

3. The homescreen's background has been changed. Viewing the background on a web viewer, its not a terrible choice ( I still prefer the image I had found of the Spoelhof center), but the big issue is that the blur is far too high. The leaves are indistinguishable. Looks like blobs or weird camo. 

4. Also on the home screen, the top two buttons have been moved down and centered. I do not think this looks as good as when they were pushed to the top.

Bongo9911 commented:
> 1. The start-up "Habit Buddy" font is not good. I would suggest finding a more professional font.

I'll put together some other examples here in a sec

> 2. The app now defaults to dark mode. I know that this is still a work in progress, but I wanted to make note of that.

That was a mistake on my part, I forgot to turn it back to default to light mode

> 3. The homescreen's background has been changed. Viewing the background on a web viewer, its not a terrible choice ( I still prefer the image I had found of the Spoelhof center), but the big issue is that the blur is far too high. The leaves are indistinguishable. Looks like blobs or weird camo.

I fixed this, but the blur is definitely higher on web browser than on the phone. I switched to a different image because I wasn't a fan of how grainy the background we currently have is.

> 4. Also on the home screen, the top two buttons have been moved down and centered. I do not think this looks as good as when they were pushed to the top.

I personally think it looks better without having lots of dead space between and instead spacing them more evenly, but we can ask the others.

andrewJamesBaker commented:
The theme is back to light which is good. The background is less blurred, but I would be curious to see how it is with a smaller blur radius. After playing around with it, I would drop it down to 1.
